rocky/katalogus/client.py

```diff
@@ -145,23 +150,32 @@ def get_cover(self, boefje_id: str) -> BytesIO:
 
 
 def parse_plugin(plugin: Dict) -> Plugin:
-    try:
-        consumes = {type_by_name(consumes) for consumes in plugin["consumes"]}
-    except StopIteration:
-        consumes = set()
+    name = plugin.get("name") or plugin["id"]
+    scan_level = None
+    if plugin["type"] != "normalizer":  # Scan levels not applicable for Normalizers
+        scan_level = SCAN_LEVEL(plugin["scan_level"])
 
+    # TODO: give normalizers a proper name in backend
+    if plugin["type"] == "normalizer":
+        name = plugin["id"].replace("_", " ").replace("kat ", "").title()
+
+    consumes = set()
     produces = set()
-    for ooi in plugin["produces"]:
-        with contextlib.suppress(StopIteration):
-            produces.add(type_by_name(ooi))
+
+    with contextlib.suppress(StopIteration):
+        if plugin["type"] != "normalizer":
+            consumes.add(
+                type_by_name(plugin["consumes"])
+            )  # TODO: No OOI type for normalizers but get consumable Boefjes objects. Refactoring needed.
```

Comment:
normalizers consume mime-types, these are listed in the normaliser manifest json, and should also include normalizer/$normalizer_name

Reply:
Yes shall I just pass the 2 items there? 
`"consumes": [
      "adr-finding-types",
      "normalizer/kat_adr_finding_types_normalize"
    ],`
I think in a next PR we can create a Boefje instance from the boefje id, we can show boefje info at Normalizer detail page

Reply:
we need to read from the manifest *and* add the later form:
 "normalizer/kat_adr_finding_types_normalize"